examples/ethereum/geth/scripts/init.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
same code is in your personal `geth-action` repo. do you need this here? or can this be used from here?

Reply:
This is useful for local testing.

Reply:
nvm, reverted